### PR TITLE
[PINOT-3464] Set the download url correctly in committing LLC segments

### PR DESCRIPTION
**Pull request by mcvsubbu**

### CODE REVIEW

File: pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java [hunk 1]
```diff
@@ -121,6 +121,11 @@ public boolean isUpdateSegmentStateModel() {
     return false;   // Default is to leave the statemodel untouched.
   }
 
+  public String generateVipUrl() {
+    return StringUtil.join("://", getControllerVipProtocol(),
```
Comment (jfim) [inline pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java]: Would be better to just write this as protocol + "://" + host + ":" + port

Comment (mcvsubbu) [inline pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java]: I copy-pasted the existing code, moving it from one class to another.

File: pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java [hunk 2]
```diff
@@ -456,4 +472,13 @@ protected void completeCommittingSegments(String realtimeTableName) {
       }
     }
   }
+
+  public static String constructDownloadUrl(String tableName, String segmentName, String vip) {
```
Comment (jfim) [inline pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java]: Should be in a more appropriate location, eg. `ControllerConf`

**Comment by jfim** [comment]
lgtm otherwise

**Pull request merged**